test/scaling_test/scaling_test.go

```diff
@@ -50,7 +50,7 @@ var _ = Describe("Scaling test", func() {
 	})
 
 	Context("Standalone deployment (S1)", func() {
-		It("scaling_test: Can Scale Up and Scale Down Standalone CR", func() {
+		It("scaling_test, integration: Can Scale Up and Scale Down Standalone CR", func() {
 
 			standalone, err := deployment.DeployStandalone(deployment.GetName(), "", "")
 			Expect(err).To(Succeed(), "Unable to deploy standalone instance ")
```

Comment:
On line 61 we check for  'testenv.GetConfiguredPeers' which also runs on MC pod. Think we need to remove that too